docs/index.rst

```diff
@@ -20,6 +20,7 @@ HDL Reference Designs
    :caption: Projects
    :hidden:
 
+   projects/ad719x_asdz/index
```

Comment:
Please do a rebase on the main branch. This list should have the following format: https://github.com/analogdevicesinc/hdl/blob/main/docs/index.rst?plain=1#L23-L26